modules/compiler/utils/source/handle_barriers_pass.cpp

```diff
@@ -347,12 +375,264 @@ struct ScheduleGenerator {
     }
   }
 
+  // Create loops to execute all the main work items, and then all the
+  // left-over tail work items at the end.
+  std::pair<BasicBlock *, Value *> makeReductionLoop(
+      compiler::utils::BarrierWithLiveVars const &barrier,
+      compiler::utils::GroupCollective const &WGC, BasicBlock *block, Value *op,
+      Value *accumulator) {
+    auto *const accTy = accumulator->getType();
+    Function *const func = block->getParent();
+
+    // Subgroup induction variables
+    Value *subgroupIVs[] = {accumulator};
+    auto *const totalSize = barrier.getTotalSize();
+
+    compiler::utils::CreateLoopOpts inner_opts;
+    inner_opts.attemptUnroll = true;
+    inner_opts.disableVectorize = true;
```

Comment:
there seens to be a bit of a mixture of naming styles here (underscores, then camelcase) - I note that above we use undescores

Reply:
yeah that's true all over the place, there's no consistency at all, and then i copy and paste a bit from somewhere else, maybe some day we should decide on something and then go through and enforce it.